contracts/deployments/gamma/base/addresses/membershipFacet.json

```diff
@@ -0,0 +1,3 @@
+{
+  "address": "0x6D0dcdC1DB42FdF90C299CCB89bC5be139aB777e"
```

Comment:
I don't know a lot about how we deploy to alpha - do we need one of these files generated for alpha as well?